backend/chainlit/config.py

```diff
@@ -51,6 +51,9 @@
 # Duration (in seconds) during which the session is saved when the connection is lost
 session_timeout = 3600
 
+# Duration (in seconds) of the user session expiry
+user_session_timeout = 1296000
```

Comment:
I would suggest adding that it's 15 days to the comment here.

Reply:
We just added a comment there to suggest `15 days`.